### PR TITLE
Apps/fix link button margin

### PR DESCRIPTION
**Pull request by manolovn**
### :goal_net: What's the goal?
Fix start margin link button when configs button shows only link.
Also I've added a change of how to put dismiss listener using a lambda

### ☑️ Checks
- [x] I updated the documentation (readmes, wikis, etc). If no need to update documentation explain why.
- [x] Tested with dark mode.
- [x] Tested with API 21.

### :test_tube: How can I test this?
- [x] 🖼️ Screenshots/Videos
image
image

- [x] Reviewed by Mistica design team

### CODE REVIEW

File: library/src/main/java/com/telefonica/mistica/callout/CalloutView.kt [hunk 1]
```diff
@@ -191,8 +193,8 @@ class CalloutView @JvmOverloads constructor(
         }
     }
 
-    fun setOnDismiss(listener: OnCalloutDismissListener) {
-        onCalloutDismissListener = listener
+    fun setOnDismiss(listener: () -> Unit) {
+        onDismissed = listener
```
Comment (mbarrben) [inline library/src/main/java/com/telefonica/mistica/callout/CalloutView.kt:197]: ❤️

File: library/src/main/java/com/telefonica/mistica/callout/CalloutView.kt [hunk 2]
```diff
@@ -283,17 +286,24 @@ class CalloutView @JvmOverloads constructor(
                 primaryButton.visibility = GONE
                 secondaryButton.visibility = VISIBLE
                 linkButton.visibility = VISIBLE
+                linkButton.layoutParams = marginStart(0f)
             }
             BUTTONS_CONFIG_LINK -> {
                 primaryButton.visibility = GONE
                 secondaryButton.visibility = GONE
                 linkButton.visibility = VISIBLE
+                linkButton.layoutParams = marginStart(-8f)
```
Comment (mbarrben) [inline library/src/main/java/com/telefonica/mistica/callout/CalloutView.kt:295]: 👍

**Review by mbarrben — APPROVED** [review]
Good job!

**Review by yamal-alm — APPROVED** [review]
nice 👏

**Review by yceballost — APPROVED** [review]
<3

**Pull request merged**